docs/presigned_urls.md

```diff
@@ -0,0 +1,354 @@
+
+&nbsp;
+## AWS Presigned URLs
+
+By default, at creation, an S3 bucket and the objects it contains are private, and only object owners are able to access or visualize them. In order to share these objects temporarily with other users, the owner can create a presigned URL, which provides a time-limited permission to view and/or download these private objects to non-owners. Presigned URLs can also be created for [uploading objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/PresignedUrlUploadObject.html) to an S3 bucket, but for use within a Chromoscope configuration file, presigned URLs will be created in order to [share private objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) temporarily.
+
+When creating a presigned URL for a specified object, the following must be provided:
+
+* Security credentials (set up in prior documentation)
+* S3 bucket name
+* Object key (i.e. relative path name of object within the bucket)
+* HTTP method (e.g. GET request for downloading)
+* URL duration (expiration time)
+
+Programmatically, a presigned URL can be created using the AWS CLI or AWS SDKs such as *boto3*. The following code snippet shows the use of the AWS CLI function `presign` to generate a presigned URL through the command line terminal, which allows for retrieval of an S3 object with an HTTP GET request:
+
+```
+aws s3 presign s3://EXAMPLE_BUCKET/EXAMPLE_OBJECT
+```
+This generates a URL for the `EXAMPLE_OBJECT`, which lives within the `EXAMPLE_BUCKET`, and expires in 3600 seconds (default). If you want to change the time duration of the URL, provide an integer value using the `--expires-in` option; the maximum length possible is 604800 seconds (7 days), as specified in the following example:
+```
+aws s3 presign s3://EXAMPLE_BUCKET/EXAMPLE_OBJECT --expires-in 604800
+```
+Example output:
+```
+https://EXAMPLE_BUCKET.s3.us-east-1.amazonaws.com/myobject?AWSAccessKeyId=AKIAEXAMPLEXXX&Expires=604800&Signature=ibOGfAovnhASUASsdasjj321dtg2s%3D
+```
+?> Note: When presigned URLs are created using the security credentials of an IAM user, the URL can last up to 7 days. When using a temporary token, the URL expires when the token expires, even if the specified time duration is longer.
+
+For further optional arguments of this AWS CLI function, see the detailed documentation [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/presign.html).
+
+One can also use AWS SDKs like *boto3* for Python to generate presigned URLs, using the `generate_presigned_url` function, using the [following script](../scripts/presigned_url_scripts/create_presigned_urls.py):
+```python
+import boto3
+from botocore.exceptions import ClientError
+
+def generate_presigned_URL(bucket_name, object_path, expiration=3600):
```

Comment:
I think, i's ok to show only the CLI approach since we want to present mainly  how to upload private data to the browser